### PR TITLE
new(blog): add blog entry for falco 0.34.0

### PR DESCRIPTION
**Pull request by LucaGuerra**
Signed-off-by: Luca Guerra <luca@guerra.sh>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind content

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

**What this PR does / why we need it**:

This tells us more about the Falco 0.34.0 release :sunglasses: 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by LucaGuerra** [comment]
This PR is work in progress since some details and links are still missing since we have more pieces of documentation to add. Some of them can only bee (pun intended) added after the previous release website has been archived.

File: content/en/blog/falco-0-34-0/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+## Updates - TL;DR 🩳
+In release v0.34.0 the community focused on addressing the following key features: 
+- Downloading and dynamically updating Falco rules at runtime
+- Shipping the brand new experimental modern eBPF probe
+- Designing more ways to catch suspicious executions
+
+## Automatic rules update 🔄
+A few questions that often come up when using Falco is how can we update rules once Falco is installed in the cluster and, how do we get updated rules from the Falco organization without having to wait for the next release? This is the first release to include [falcoctl](https://github.com/falcosecurity/falcoctl) with an out of the box solution to do exactly that!
+
+When using the new Falco Helm Chart 3.0.0 rules will automatically be updated from the official repository. To learn more about this feature and how to configure it read [xxx] links. Likewise, when using a SystemD based install you can ([xxx] link) configure Falco to automatically update rules.
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-34-0/index.md:29]: * link to systemd: https://github.com/falcosecurity/falco-website/pull/799

File: content/en/blog/falco-0-34-0/index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+### Why a new probe? 👽
+The old probe supported old kernels (>=4.14) that can not take advantage of the new shiny eBPF features. While it would be great to have only one probe that works for every kernel version, recent features change (and simplify!) the way we write, maintain and deploy the code so deeply that a new fresh probe is the most reasonable solution. In order to leverage these recent eBPF improvements and use the new probe you will need a kernel version >= 5.8.
+
+### Modern eBPF in action 🎬
+Try it now by following these instructions [xxx link]
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-34-0/index.md:38]: * link to the documentation section https://github.com/falcosecurity/falco-website/pull/800

File: content/en/blog/falco-0-34-0/index.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+---
+
+![](images/falco-honeybee.png)
+
+Dear community, today we are delighted to announce the release of Falco 0.34.0  🎉!
+
+A big thank you to the community for helping get the latest release out. The Falco community is thrilled about this release and cannot wait to share the latest goodies. Check out the newest features from this most recent launch to learn more ⬇️. To read a more detailed account of the release, see [v0.34.0 in the changelog]().
+
+## What’s new? 🆕
+In this release we saw more than xxx pull requests across the repos of Falco and its libraries. Thank you to our maintainers and contributors, as this would not happen without your support, dedication, and contribution.
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-34-0/index.md:18]: add final number at time of release

File: content/en/blog/falco-0-34-0/index.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+images:
+  - images/falco-hobeybee-featured.png
+  - images/falco-honeybee.png
+---
+
+![](images/falco-honeybee.png)
+
+Dear community, today we are delighted to announce the release of Falco 0.34.0  🎉!
+
+A big thank you to the community for helping get the latest release out. The Falco community is thrilled about this release and cannot wait to share the latest goodies. Check out the newest features from this most recent launch to learn more ⬇️. To read a more detailed account of the release, see [v0.34.0 in the changelog]().
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-34-0/index.md:15]: add link to the changelog

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**